libs/pdi-scanner/src/rust/client.rs

```diff
@@ -649,6 +659,7 @@ impl<T> Client<T> {
         &mut self,
         calibration_type: DoubleFeedDetectionCalibrationType,
     ) -> Result<()> {
+        self.set_double_feed_detection_mode(DoubleFeedDetectionMode::Disabled)?;
```

Comment:
Some docs from PDI indicated that it was a "common sequence" to disable DFD before calibrating, so I replicated that here to be safe